### ISSUE TITLE
Set min width on buttons

### ISSUE THREAD

**Issue by laurent22** [laurent22/joplin#15959]
<img width="376" height="219" alt="Image" src="https://github.com/user-attachments/assets/651699e0-b325-4aa7-9a38-61cb09bc4890" />

We should set a min width on the button that is larger than what is on this screenshot for UX and accessibility purposes. Is there a general recommendation on what a good width should be ?
